code/modules/projectiles/projectile/special/plasma.dm

```diff
@@ -32,18 +22,18 @@
 			return -1
 
 /obj/item/projectile/plasma/adv
-	damage = 28
+	damage = 7
 	range = 5
 	mine_range = 5
 
 /obj/item/projectile/plasma/adv/mech
-	damage = 40
+	damage = 10
 	range = 9
 	mine_range = 3
 
 /obj/item/projectile/plasma/turret
 	//Between normal and advanced for damage, made a beam so not the turret does not destroy glass
 	name = "plasma beam"
-	damage = 24
+	damage = 6

```

Comment:
This shouldn't be nerfed, there are no alternatives for the aux base, and isn't spammable

Reply:
Yes, I wanted them to be KA turrets, but 1)KAs are weird and I did not know to make a turret fire KA shots 2) KA shots cannot go through windows. 3)KA shots do not have enough range

The turrets are for damage more than mining (you cannot fire the turrets manually). Also, unless Xenos or blobs sit themselves at or near the Aux Base, AND somehow allows someone to set up the turrets, I do not see them ever being a threat to them.

As the Aux Base turrets do not fall under the terms of your complaint, I would request this particular change to be reverted. My guess is that the OP went down the list of all the plasma project tiles and indiscriminately modified them all.